__tests__/app.test.js

```diff
@@ -54,104 +55,140 @@ describe("GET /api/topics", () => {
+        expect(body.msg).toBe("404: Article not found");
       });
-    });
-    it("404: responds with a bad request, no article exists", () => {
-      return request(app)
-        .get("/api/articles/205345345/comments")
-        .expect(404)
-        .then(({ body }) => {
-          expect(body.msg).toBe("404: Article not found");
-        });
-    });
-    it("400: Invalid article_id", () => {
-      return request(app)
-        .get("/api/articles/sdfsdfsdf/comments")
-        .expect(400)
-        .then(({ body }) => {
-          expect(body.msg).toBe("400: Invalid article_id");
-        });
-    });
-    it("returns an empty array of comments when article has no comments", () => {
-      return request(app)
-        .get("/api/articles/7/comments")
-        .expect(200)
-        .then(({ body }) => {
-          expect(body.comments).toBeInstanceOf(Object);
-          expect(body.comments.length).toBe(0);
-        });
-  })
-    
-  })
+  });
+  it("400: Invalid article_id", () => {
+    return request(app)
+      .get("/api/articles/sdfsdfsdf/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("400: Invalid article_id");
+      });
+  });
+  it("returns an empty array of comments when article has no comments", () => {
+    return request(app)
+      .get("/api/articles/7/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeInstanceOf(Object);
+        expect(body.comments.length).toBe(0);
+      });
+  });
+});
+
+describe("POST /api/articles/:article_id/comments", () => {
+  it("201 - responds with a newly created comment ", () => {
+    const newComment = {
+      body: "Yo wadup man!",
+      username: "butter_bridge",
+    };
+    return request(app)
+      .post("/api/articles/7/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment.body).toBe("Yo wadup man!");
+        expect(body.comment.author).toBe("butter_bridge");
+        expect(body.comment.votes).toBe(0);
+        expect(typeof body.comment).toBe("object");
+        expect(body.comment).toHaveProperty("comment_id");
+        expect(body.comment).toHaveProperty("body");
+        expect(body.comment).toHaveProperty("votes");
+        expect(body.comment).toHaveProperty("author");
+        expect(body.comment).toHaveProperty("created_at");
+        expect(body.comment.comment_id).toBeGreaterThan(0);
+      });
+  });
+  it("404: responds with a bad request, no article exists", () => {
+    return request(app)
+      .get("/api/articles/205345345/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("404: Article not found");
+      });
+  });
+  it("400: Invalid article_id", () => {
+    return request(app)
+      .get("/api/articles/sdfsdfsdf/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("400: Invalid article_id");
+      });
+  });
```

Comment:
Excellent testing so far - consider these instances:
201 - ignore unnecessary properties in the sent object
400 - missing required fields (i.e. no body or no username)
404 - username does not exist